catroid/src/main/java/org/catrobat/catroid/utils/TrackingUtil.java

```diff
@@ -61,7 +61,7 @@ private TrackingUtil() {
 	}
 
 	public static void trackCreateProgram(String projectName, Boolean landscapeMode, boolean exampleProgram) {
```

Comment:
just looking at the div below, i see the very same if clauses about 50 times.
why not implement a trackerClass that is then instantiated ONCE based on the build flag at startup, either as activeTracker, or as "blackHole" or whatever thats just swallowing tracking calls?